jax/_src/config.py

```diff
@@ -92,15 +93,27 @@ class Config:
   def __init__(self):
     self.values = {}
     self.meta = {}
-    self.FLAGS = NameSpace(self.read, self.update)
     self.use_absl = False
     self._contextmanager_flags = set()
     self._update_hooks = {}
 
+  def __getattr__(self, name):
```

Comment:
Can you do this via the existing `jax._src.deprecations.deprecation_getattr` utility instead?

Reply:
Oh, I guess not, because these aren't module-level deprecations...

Reply:
No, I think the custom deprecation code is fine.